index.html

```diff
@@ -3979,24 +3979,45 @@ <h3>Receiving and parsing content</h3>
       </li>
     </ol> <!-- parsing NDEF text record -->
   </p>
-  <p class="note" data-link-for="NDEFRecord">
-    Using the <a data-cite="encoding#encoder">encoder</a>, it is only possible
-    to encode as UTF-8, unless you do the encoding manually
-    and use the "{{opaque}}" {{recordType}}, which
-    allows you to write any opaque data.
-    <br><br>
-    When you write a <a>string</a> to the "{{text}}"
-    {{recordType}}, it will be written as UTF-8.
-    External applications have the ability to additionally encode
-    this field as UTF-16BE, but that is transparent from the use
-    of the {{NFCReader}} which will always return the data
-    as a <a>string</a>.
-    <br><br>
-    As external applications may have encoded the "{{opaque}}"
-    {{recordType}} using a different
-    encoding, which can be decoded using [[[ENCODING]]] if you know
-    the encoding.
-  </p>
+  <aside class="note" data-link-for="NDEFRecord">
+    <p>
+      The Unicode standard defines multiple encodings like UTF-8, UTF-16 and UTF-32.
+      UTF-8 is the preferred encoding on the web, and it has the advantage
+      that it is endianness agnostic, as code points are represented in single bytes.
+    </p>
+    <p>
+      The NDEF text records allow the text to be encoded as either UTF-8 and UTF-16.
+      Generally, it is preferred to use UTF-8, but integration with existing systems
```

Comment:
Core NFC uses UTF-16 when creating a text record so we may want to skip this part.

Reply:
On the web UTF-8 is definitely preferred. How do you want to rephrase?

Reply:
Maybe something like "UTF-8 is the preferred format on the Web. This is why text records in Web NFC defaults to 
UTF-8"